### ISSUE TITLE
Getting ManifestMerger2 MergeFailureException on basic project

### ISSUE THREAD

**Issue by DarioAhdoot** [Vigi0303/fat-aar-plugin#16]
As mentioned in the other Issue, I'm having Manifest Merging issues, even on Android Gradle Plugin 2.2.3 and with build cache disabled.

I've attached a minimum reproducible example:
[FatAarExample.zip](https://github.com/Vigi0303/fat-aar-plugin/files/1521570/FatAarExample.zip)

Here we have three Library projects (LibraryA, LibraryB, and LibraryC), and a fourth (FatLibrary) which tries to embed them.

Import this into Android Studio. Both Syncing and Building will fail with the given exception.

Please let me know if you have any questions.

**Comment by vigidroid** [comment]
To add this line to the manifest node in your main manifest file does the trick.
```xml
xmlns:tools="http://schemas.android.com/tools"
```

I will try to fix it someday.

**Comment by DarioAhdoot** [comment]
Thanks, that fixed the minimum reproducible example I provided, but we're still seeing the issue with our actual project. I'll have to do some more investigating. Our manifests are basically identical, as are our build.gradle files. Frustrating.

**Comment by vigidroid** [comment]
@Diarrhio yes it's frustrating. In the source code of android plugin, the handling of manifest merge part is such complicated.
You can post the full stacktrace of your project, to try the luck.
